### ISSUE TITLE
Services sync dialogue UI: allow user to show and hide sync details

### ISSUE THREAD

**Issue by maprehensive** [opendatakit/tool-suite-X#332]
When a sync is occurring the dialogue box can show a lot of rapidly changing sync details text. This can be uninformative and districting. 

Allow users to hide the sync details text by adding a Show / Hide button to the dialogue box . 

Note that color and layout updates are detailed in #331 

![Sync dialogue - details hidden](https://user-images.githubusercontent.com/1215441/153857939-b6accbf8-e5b3-447e-9168-f9710c8bd4d8.png)
![Sync dialogue - details shown](https://user-images.githubusercontent.com/1215441/153857942-a35c0899-a4a1-4aa5-8b0a-69d603f06cd6.png)

**Comment by OlaitanAdesina** [comment]
Hello, I would like to work on this issue.

**Comment by wbrunette** [comment]
@OlaitanAdesina I have assigned the issue to you

**Comment by BlueSkiez-dev** [comment]
Hi, I would like to work on this issue

**Comment by cvemann** [comment]
Is this issue open to simultaneous people? If yes, I'd also like to work on it to improve it's UX @maprehensive

**Comment by ghost** [comment]
Hello @maprehensive @wbrunette I would love to work on the UI for this issue for my outreachy contribution. Thanks

**Comment by G-ann** [comment]
Hello, I would love to work on this, if it's still open @maprehensive

**Comment by maprehensive** [comment]
@wbrunette – can I open this issue to Outreachy applicants? Will probably need guidance on the technical implementation.

**Comment by caxrii** [comment]
I would like to work on this?

**Comment by maprehensive** [comment]
@BlueSkiez-dev @Cveman1 @AkorOG @G-ann Yes, please go ahead and work on this. This is a technical piece, and will require some input from @wbrunette

**Comment by caxrii** [comment]
@maprehensive @wbrunette what are the technical input required for this.

**Comment by wbrunette** [comment]
You will need to check out the Services code and find the appropriate place to update colors. https://developer.android.com/

**Comment by salmasabo** [comment]
I would like to work on this issue

**Comment by BlueSkiez-dev** [comment]
@salmasabo you can go ahead

**Comment by JessicaItepu** [comment]
..

**Comment by cvemann** [comment]
Hi @maprehensive. Trust you're doing good.

I have adjusted the UI of the Services dialogue UI to allow users to show and hide sync details, in a more modern, cleaner and clutter-free design. 

Here it is;
![image](https://user-images.githubusercontent.com/80982139/200208440-c4032d67-b0a0-4061-bf9b-e5ce53381414.png)

Although I believe I have fixed the issue of the 'rapidly changing sync details text' in my [earlier contribution](https://github.com/odk-x/tool-suite-X/issues/331#issuecomment-1286304957) to #331. This new iteration aims to design a modern dialogue UI with less clutter. 
Users can hide and show sync details with a simple text link button, and an 'expand' arrow for visual cues to the user.
I'm still in line with the [ODK-X Design Guidelines](https://docs.google.com/presentation/d/1Ol7YhG4WPLOWCoQZUrKHLPguHSiDz8RC3m8dNyTGGLo/edit#slide=id.g13cf9ff5306_0_0).

View the prototype here;

https://user-images.githubusercontent.com/80982139/200209010-bb72ae63-f782-4dbd-a25a-a164a99d3568.mp4



Experience the prototype [here](https://www.figma.com/proto/UIozO1danB9PxHacQZXVZL/ODK-X-UI-Components---Ukpong-Godswill?page-id=219%3A526&node-id=219%3A819&viewport=887%2C673%2C0.58&scaling=scale-down&starting-point-node-id=219%3A819), while I figure out how to put this up on Android Studio.

**Comment by cvemann** [comment]
This #331 also solves the same problem. @Redeem-Grimm-Satoshi

**Comment by Redeem-Grimm-Satoshi** [comment]
Yes @Cveman1 It does.

**Comment by maprehensive** [comment]
PR https://github.com/odk-x/services/pull/256 raised, needs updating

**Comment by maprehensive** [comment]
I've reviewed this on the outreach-2023 branch and it looks good. Works as designed. 

@Cveman1 will you review this as well?
